aten/src/ATen/native/transformers/cuda/attention.cu

```diff
@@ -819,7 +823,8 @@ std::tuple<Tensor, Tensor, int64_t, int64_t, Tensor> _flash_attention_forward(
     const int64_t max_seqlen_batch_k,
     double dropout_p,
     bool is_causal,
-    bool return_debug_mask) {
+    bool return_debug_mask,
+    c10::optional<double> scale) {
```

Comment:
Why make this optional at all?

Reply:
Never mind, we need that to deal with FC issues...

But can we unify the scale calculation into a single location somewhere? Say via a helper function?